docs/Config_Reference.md

```diff
@@ -2457,14 +2457,20 @@ information.
 #tachometer_ppr:
 #tachometer_poll_interval:
 #   See the "fan" section for a description of the above parameters.
-#sensor_type:
-#sensor_pin:
 #control:
+#   In addition to "watermark" and "pid", temperature_fan has an additional
+#   "linear" control that sets the fan speed to a value between min_speed
+#   and max_speed, starting from a temperature offset below the target_temp.
+#linear_offset: 10.0
+#   on "linear" control, this sets the temperature offset below the
+#   target_temp where the linear scaling starts
```

Comment:
Not clear to me from the explanation you gave.
Do you maybe mean
"this sets the temperature offset which causes the fan to spin at max or min speed, depending on the sign of the temperature difference"

Reply:
Or also, more textbook, less user friendly:
"this sets the temperature offset which causes the maximum compensation (the fan speed is set to one of the extremes)"